pkg/applicationserver/io/packages/loradms/v1/package.go

```diff
@@ -85,7 +85,7 @@ func (p *DeviceManagementPackage) HandleUp(ctx context.Context, def *ttnpb.Appli
 		join := m.JoinAccept
 		loraUp := &objects.LoRaUplink{
 			Type:      objects.JoiningUplinkType,
-			Timestamp: float64Ptr(float64(ttnpb.StdTime(join.ReceivedAt).UTC().Unix())),
+			Timestamp: float64PtrOfTimestamp(join.ReceivedAt),
```

Comment:
`ttnpb.StdTime(x).UTC()` panics if `x` is `nil`, and it is `nil` for simulated uplinks.